packages/@okta/vuepress-site/docs/reference/test.md

```diff
@@ -0,0 +1,9 @@
+Hello world!
+
+-=OKTA_REPLACE_WITH_WIDGET_VERSION=-
+
+-=OKTA_REPLACE_WITH_WIDGET_VERSION=- "-=OKTA_REPLACE_WITH_WIDGET_VERSION=-"
+-=OKTA_REPLACE_WITH_TEST_JUNK=-
+```
+-=OKTA_REPLACE_WITH_WIDGET_VERSION=-
+```
```

Comment:
We should only have these at `/packages/@okta.vuepress-site/test_page/index.md`

Reply:
D'oh!  This was my early test file.  I even checked _twice_ to make sure I hadn't included it.  Sigh.